docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -430,22 +430,20 @@ By default the controller redirects all requests to an existing service that pro
 
 ### Rate limiting
 
-These annotations define a limit on the connections that can be opened by a single client IP address.
-This can be used to mitigate [DDoS Attacks](https://www.nginx.com/blog/mitigating-ddos-attacks-with-nginx-and-nginx-plus).
+These annotations define limits on connections and transmission rates.  These can be used to mitigate [DDoS Attacks](https://www.nginx.com/blog/mitigating-ddos-attacks-with-nginx-and-nginx-plus).
 
-* `nginx.ingress.kubernetes.io/limit-connections`: number of concurrent connections allowed from a single IP address.
-* `nginx.ingress.kubernetes.io/limit-rps`: number of connections that may be accepted from a given IP each second.
-* `nginx.ingress.kubernetes.io/limit-rpm`: number of connections that may be accepted from a given IP each minute.
-* `nginx.ingress.kubernetes.io/limit-rate-after`: sets the initial amount after which the further transmission of a response to a client will be rate limited.
-* `nginx.ingress.kubernetes.io/limit-rate`: rate of request that accepted from a client each second.
+* `nginx.ingress.kubernetes.io/limit-connections`: number of concurrent connections allowed from a single IP address. A 503 error is returned when exceeding this limit.
+* `nginx.ingress.kubernetes.io/limit-rps`: number of requests accepted from a given IP each second. The burst limit is set to 5 times the limit. A 503 error is returned when exceeding this limit.
+* `nginx.ingress.kubernetes.io/limit-rpm`: number of requests accepted from a given IP each minute. The burst limit is set to 5 times the limit. A 503 error is returned when exceeding this limit.
+* `nginx.ingress.kubernetes.io/limit-rate-after`: initial number of kilobytes after which the further transmission of a response to a given connection will be rate limited.
+* `nginx.ingress.kubernetes.io/limit-rate`: number of kilobytes per second allowed to send to a given connection.  The zero value disables rate limiting.
+* `nginx.ingress.kubernetes.io/limit-whitelist`: client IP source ranges to be excluded from rate-limiting. The value is a comma separated list of CIDRs.
 
-You can specify the client IP source ranges to be excluded from rate-limiting through the `nginx.ingress.kubernetes.io/limit-whitelist` annotation. The value is a comma separated list of CIDRs.
+If you specify multiple annotations in a single Ingress rule, limits are applied in the order `limit-connections`, `limit-rps`, `limit-rpm`.
```

Comment:
the order (in code) is `limit-connections`, `limit-rpm`, `limit-rps` https://github.com/kubernetes/ingress-nginx/blob/f5c5e122523a8ebe6f862def6e9db396937e2225/internal/ingress/controller/template/template.go#L591

Reply:
of course, mistyped.